packages/create/package.json

```diff
@@ -37,12 +37,17 @@
         "@sap-ux/app-config-writer": "workspace:*",
         "@sap-ux/cap-config-writer": "workspace:*",
         "@sap-ux/cards-editor-config-writer": "workspace:*",
+        "@sap-ux/inquirer-common": "workspace:*",
         "@sap-ux/logger": "workspace:*",
         "@sap-ux/mockserver-config-writer": "workspace:*",
         "@sap-ux/project-access": "workspace:*",
+        "@sap-ux/system-access": "workspace:*",
+        "@sap-ux/ui5-config": "0.22.3",
```

Comment:
please use `workspace:*` also for ui5-config

Reply:
Changed in cd1b20bbdddd1c595d925b5276ce529ca7ab8fa5